doc/changes/changes_15.0.0.md

```diff
@@ -0,0 +1,26 @@
+# Virtual Schema Common Java 15.0.0, released 2021-??-??
+
+Code name: 
+
+In release 15.0.0, we optimized adapter loading by limiting the number of `Virtual Schema Adapter` to be loaded to one.
```

Comment:
That's not very clear to me. 
It could also mean that you can only have an oracle **or** a PostgreSQL virtual schema in an Exasol database.

```suggestion
In release 15.0.0, we changed the deployment so that each dialect gets its own jar. By that, we could remove one service loader level.
```

Maybe you can also describe what changed directly for users. I guess one property got obsolte.

Reply:
The dialect property is obsolete, but we haven't removed yet (I not in this PR for sure). I'll add a comment though.

Reply:
BTW, the service loader level is not removed, it's still there, we have 2 levels. The difference is that now we don't need a registry as we only allow one dialect.